### ISSUE TITLE
Use `aria-current="true"` instead of `<strong>` for `<nav>`

### ISSUE THREAD

**Issue by fritzmg** [contao/contao#8581]
### Affected version(s)

N/A

### Description

I wanted to discuss whether it makes sense to use `<span aria-current="true">` instead of `<strong>` for the currently active navigation item.

`<strong>` in HTML automatically gets the [ARIA `strong` role](https://www.w3.org/TR/wai-aria-1.2/#strong) - but that only describes

> Content that is important, serious, or urgent.

and does not necessarily signify that this is the active navigation item.

Whereas [`aria-current="true"`](https://www.w3.org/TR/wai-aria-1.2/#aria-current)

> Indicates the [element](https://www.w3.org/TR/wai-aria-1.2/#dfn-element) that represents the current item within a container or set of related elements.

**Comment by asaage** [comment]
From the current contao-demo:
```<li class="active"><strong class="active" aria-current="page">Link elements</strong></li>```
I'd prefer ```span``` instead of ```strong``` but definitely keep ```class="active"``` and doublecheck wheather aria-current should be "true" or "page"

**Comment by fritzmg** [comment]
> but definitely keep `class="active"`

Yes, the classes will definitely be kept.

> and doublecheck wheather aria-current should be "true" or "page"

The spec doesn't seem to be super clear about that. It states that "page" should be used for the active page of a pagination - which would not apply to a navigation in my opinion.

**Comment by fritzmg** [comment]
That comment refers to a navigation with the `tree` role, which is only supposed to be applied to vertical navigation trees, as far as I understand the spec.

**Comment by zoglo** [comment]
We are already using `aria-current="page"` as opposed to how it should be:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-current
> In a breadcrumb list, when a link within a set of pagination links is styled to indicate the user is currently on that page, aria-current="page" should be set on that link

What's wrong with leaving it at that 🤔 ?

**Comment by fritzmg** [comment]
The [specification](https://w3c.github.io/aria/#aria-current) says

> A `page` token used to indicate a page within a set of pages, where the element is visually styled to represent the current page.

> For example, aria-current="page" can be used in a navigation [tree](https://w3c.github.io/aria/#tree) to indicate which page is currently displayed

> _page_ Represents the current page within a set of pages.

But then there is (unofficial) examples like [this](https://a11ysupport.io/tests/html/aria/aria-current.html), where they use `location` for the breadcrumb (which I guess kind of makes sense in the context of a breadcrumb) and `page` for a general navigation. Other unofficial examples use `true` or `page`.

So it is not 100% clear to me that what the intended purpose of the _page_ token is supposed to be used for.

But then again, the [APG example](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/) does use `aria-current="page"` - so I guess if it's safe to use for the breadcrumb, it probably also can be applied to any other form of web page navigation.
